nevergrad/benchmark/experiments.py

```diff
@@ -43,6 +43,54 @@
 # fmt: off
 
 
+def _positive_sum_float(data: np.ndarray) -> float:  # This one is Boolean.
+    if not isinstance(data, np.ndarray):
+        raise ValueError(f"Unexpected inputs as np.ndarray, got {data}")
+    return float(np.sum(data))
+
+
+def _positive_diff_float(data: np.ndarray) -> float:
+    if not isinstance(data, np.ndarray):
+        raise ValueError(f"Unexpected inputs as np.ndarray, got {data}")
+    return float(np.sum(data[::2]) - np.sum(data[1::2]))
+
+
+def _positive_second_diff_float(data: np.ndarray) -> float:
+    if not isinstance(data, np.ndarray):
+        raise ValueError(f"Unexpected inputs as np.ndarray, got {data}")
+    return float(2 * np.sum(data[1::2]) - 3 * np.sum(data[::2]))
+
+
+def _ball_float(data: np.ndarray) -> float:
+    if not isinstance(data, np.ndarray):
+        raise ValueError(f"Unexpected inputs as np.ndarray, got {data}")
+    return float(np.sum(np.square(data))) - float(len(data)) - float(np.sqrt(len(data)))  # Most points violate the constraint.
```

Comment:
that's way too much repeated code.

You should go with something like
```python
class _Constraint

    def __init__(self, name: str) -> None:
        ...

    def __call__(self, data: np.ndarray) -> float:
        ...
```
it will still be a lot of ifs, but that would be 2 lines per case instead of 6

Reply:
Done.